transcom/mymove
# INT B-20294 CSS fixes for ppm packet download tag

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20294)

## Summary

There were some CSS issues that were popping up when the TOO and SC were viewing a PPM shipment that had the `Tag` showing the `PPM PACKET READY FOR DOWNLOAD`. See example here from the Agility ticket:
![953403](https://github.com/user-attachments/assets/50d82f0a-55c5-4ba9-b158-5be607efa7df)

After digging around in `ShipmentDisplay.jsx` I realized that they were using the atrocious `display: grid` with a wombo combo of using flexbox and that just wouldn't do and it seemed they conflicted with each other on render. 

When it comes to flexbox, we have to arrange things in a certain way to get them to render a certain way. In this case, they were rendering the "PPM" and the shipment number in one column, and then had the PACKET tag in another column with a ton of negative left margin on it. No thank you.

This PR does the following:
- arranges divs to where the shipment type & tags are in a single div and then the shipment number is in another div underneath it - wrapped up tidy in a flexbox
- addressed some other CSS annoyances I spotted during analysis
- fixed vertical alignment of shipment type & shipment number in customer side when viewing the simplified shipment list
- fixed vertical alignment of the move code in the `CustomerHeader` component


### How to test

1. Create a PPM as a customer
2. Approve the PPM as a SC
3. Bounce between the SC & TOO when viewing the move/shipment and make sure that the `PACKET READY FOR DOWNLOAD` does not bleed over the shipment number

### Screenshots
Finished product
![Screenshot 2024-07-30 at 11 26 31 AM](https://github.com/user-attachments/assets/2edfe75a-9d87-44ea-869e-c108b0c71d63)


**Fixing alignment of PPM shipment display & shipment number** (note: this is centered in the middle just to show you how off they were - added back `justify-content: space-between` after taking the screenshot)
Before
![Screenshot 2024-07-30 at 9 55 04 AM](https://github.com/user-attachments/assets/40391f00-70ee-488b-acad-aaa808186d10)
After
![Screenshot 2024-07-30 at 9 55 15 AM](https://github.com/user-attachments/assets/6fe679d3-d328-4f1f-a584-73949428b539)

**Fixing non-vertically centered move code in office user header**
Before
![Screenshot 2024-07-30 at 10 40 31 AM](https://github.com/user-attachments/assets/064eccc3-bfca-4e80-a811-2e5eab7ca635)
After
![Screenshot 2024-07-30 at 10 40 43 AM](https://github.com/user-attachments/assets/e0c6e115-77c2-4107-b6e1-f7d18fb6a352)
